src/view/com/modals/ServerInput.tsx

```diff
@@ -91,7 +91,7 @@ export function Component({onSelect}: {onSelect: (url: string) => void}) {
             <TextInput
               testID="customServerTextInput"
               style={[pal.borderDark, pal.text, styles.textInput]}
-              placeholder="e.g. https://bsky.app"
+              placeholder={_(msg`e.g. https://bsky.app`)}
```

Comment:
We don't want to localize a URL

Reply:
I included it in the localization because of `e.g.`. Is it okay to wrap the URL in `${}`?